registration_runbook.md

```diff
@@ -1,57 +1,42 @@
 # Registration/Attendees
 
-Use this template to write up a runbook for your area of concern.
+This runbook is focused on Registration / Ateendees and what you need to prep for taking on this role! There are a several key elements that are needed to be done before, during, and after the conference to ensure a succesful completion of the Registration role. In a nutshell, our job is to ensure that attendees can register, attend, and are reminded of the awesome DevOps exerpeince they will experience at DevOpsDays Austin. Please read below to learn more!
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+The role of registration is centered around the idea of selling / supplying DevOpsDays tickets to our attendees + vendors + speakers. The registration team needs to collaborate and work with other organizer teams (specifically Website, Swag, and Volunteers) since multiple areas are tightly coupled with the registration organizer team.
 
 ### General Tasks
 
 Can you name typical tasks? For example:
 
-* Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
+* Setup the Ticket Registratiation Tool (Verbage, Finalize Ticket Prices, Vendor Codes, etc.).
+* Work with Website Organizer Team to coordinate when we should promote ticket sales (Early Bird, Normal Sale, etc.).
```

Comment:
Dates (the when) is a core question, but you are coordinating with website to get the links up